README.md

```diff
@@ -5,7 +5,128 @@
 
 ## Getting started
```

Comment:
We could probably have a first section line in https://github.com/gsoft-inc/gsoft-authentication-clientcredentialsgrant where we briefly outline the different packages and what they include and the lower in the readme we go into more details on each with examples.